### DISCUSSION TITLE
Cant Delete a Relationship Column

### DISCUSSION THREAD

**Discussion by Jake-qp** [Budibase/budibase#1407] [Help] [answered]
I've tried to delete a column that type: Relationship, but there is no option to do so. How do I delete an existing relationship column?
<img width="334" alt="Budibase_Builder" src="https://user-images.githubusercontent.com/53414466/115955304-ae9c7800-a4c3-11eb-9046-988ec450101e.png">

**Comment by joebudi** [comment, marked as answer]
Hi @Jake-qp . i just test this and it seems to be a bug. I will raise an issue. Thank you for letting us know.

**Reply by joebudi** [reply]
The issue can be found:
#1408
